src/windows.rs

```diff
@@ -7,7 +7,7 @@
 //! This module largely exists to integrate into libstd itself where winapi is
 //! not currently available.
 
-#![allow(bad_style, dead_code)]
+#![allow(bad_style, dead_code, unused)]
```

Comment:
It kinda feels like we should `cfg` this code properly rather than allowing unused and dead_code but that's an existing issue, so fair enough.

Reply:
I agree. I can clean this up in another PR maybe? I just poked through this and I think I've convinced myself that I can hack on this in this repo and not break rust-lang/rust's module use of it. I was worried about that somehow relying on visibility.

Reply:
I took a crack at this and decided I don't have the patience to untangle all the ways that things get used to not used based on the numerous cfgs that this crate is compiled with.